SecurityManager/Core.UnitTests/Domain/SecurityManagerPrincipalTests/Common.cs

```diff
@@ -139,6 +139,10 @@ public void Serialization ()
 
       Assert.That(deserializedPrincipal.Substitution.ID, Is.EqualTo(principal.Substitution.ID));
       Assert.That(deserializedPrincipal.Substitution, Is.Not.SameAs(principal.Substitution));
+
+      Assert.That(() => deserializedPrincipal.GetTenants(true), Throws.Nothing);
+      Assert.That(() => deserializedPrincipal.GetTenants(false), Throws.Nothing);
+      Assert.That(() => deserializedPrincipal.GetActiveSubstitutions(), Throws.Nothing);
```

Comment:
```suggestion
      Assert.That(() => deserializedPrincipal.GetTenants(true), Throws.Nothing);
      Assert.That(() => deserializedPrincipal.GetTenants(false), Throws.Nothing);
      Assert.That(() => deserializedPrincipal.GetActiveSubstitutions(), Is.Not.Empty);
```

There's active substitutions based on the line 126. It would be nice to also check for Not-Empty on GetTenants. I would assume atleast one of those methods also returns a non-empty value. Didn't check, though.